src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -1046,21 +1046,17 @@ public RunningBackup OnGoingBackup(long taskId)
 
         public void HandleDatabaseValueChanged(string type, RawDatabaseRecord record, object changeState)
         {
-            switch (type)
-            {
-                case nameof(UpdateResponsibleNodeForTasksCommand):
-                    UpdateConfigurations(record.PeriodicBackups);
-                    break;
+            UpdateConfigurations(record.PeriodicBackups);
 
-                case nameof(DelayBackupCommand):
-                    var state = (DelayBackupCommand.DelayBackupCommandState)changeState;
-                    if (_periodicBackups.TryGetValue(state.TaskId, out var periodicBackup) == false)
-                        throw new InvalidOperationException($"Backup task id: {state.TaskId} doesn't exist");
+            if (type == nameof(DelayBackupCommand))
```

Comment:
This isn't an issue for this command since we'll always run this regardless of the already executed index.
https://github.com/ravendb/ravendb/blob/d35bc79d3a4cbd7320eee1f0924931f43619fcbe/src/Raven.Server/Documents/DocumentDatabase.cs#L1516